src/react/deckgl.js

```diff
@@ -118,141 +129,19 @@ export default class DeckGL extends React.Component {
     this.props.onWebGLInitialized(gl);
 
     // Note: avoid React setState due GL animation loop / setState timing issue
-    this.layerManager = new LayerManager({gl});
+    this.layerManager = new LayerManager({gl, canvas});
```

Comment:
The link was just to show that `canvas` is already available as a member on the `gl` context that we are already passing as a parameter, so it might be unnecessary to add the `canvas` parameter.

But perhaps we want to handle the case where the event canvas is different from the canvas that the context was created on.